src/gpu/intel/gemm/jit/generator/pieces/copy.cxx

```diff
@@ -247,7 +247,7 @@ void Generator<hw>::copyExecute(CopyPlan &&plan, CommonState &state)
             mov(1, savedFlags[i], FlagRegister(i));
 
     // Workaround probable scheduling / hw issue
-    sync(SyncFunction::nop, SWSB<uint32_t>(1));
+    sync(SyncFunction::nop, 1);
```

Comment:
I don't think this is having the effect you intended -- 1 is treated as the SIMD size for sync (ignored).

For XeLP, the equivalent here would be `SWSB<AllPipes>(1)`. But if you're not seeing any issues on XeLP, maybe you can do `if (hw > HW::Gen12LP) sync(SyncFunction::nop, SWSB<uint32_t>(1));`